### PR TITLE
Delete specific get

### PR DESCRIPTION
**Pull request by MilanBrkic**
Changed code from feedback given in previous Pull Request. Added GET for specific issue (/api/issues/:id), for viewing specific file (api/issues/uploads/:link), and downloading it (api/issues/uploads/download/:link) (doesn't work on postman, but works on browser). Added DELETE (/api/issues/:id) for deleting specific issue.
Specific GET, specific DELETE and DELETE for deleting all (new route: /api/issues/delete/all) tested.

### CODE REVIEW

File: test/testDelete.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                    .delete('/api/issues/delete/all')
+                    .end((err, res) => {
+                        res.should.have.status(200);
+                        res.body.should.be.an('object');
+                        res.body.should.have.property('ok').and.to.be.eql(1);
+                        res.body.should.have.property('n').and.to.be.eql(1);
+                        res.body.should.have.property('deletedCount').and.to.be.eql(1);
+                        done();
+                    })
+            })
```
Comment (rastasheep) [inline test/testDelete.js:42]: It would be nice to check if no issues exists after deletion is done

File: test/testDelete.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                    .end((err, res) => {
+                        res.should.have.status(200);
+                        res.body.should.be.an('object');
+                        res.body.should.have.property('deleted').and.to.be.an('object');
+                        var deleted = res.body.deleted;
+                        deleted.should.have.property('title').and.to.be.eql(result.title);
+                        deleted.should.have.property('user').and.to.be.eql(result.user);
+                        deleted.should.have.property('text').and.to.be.eql(result.text);
+                        deleted.should.have.property('closed').and.to.be.eql(result.closed);
+                        done();
```
Comment (rastasheep) [inline test/testDelete.js:94]: Same as above, your test checks if response is ok (which is fine) but it doesn't mean that issue is deleted for sure

File: test/testDelete.js (hunk 1 above)
Comment (MilanBrkic) [reply test/testDelete.js:42]: Yes, I totally forgot about that.

**Comment by MilanBrkic** [comment]
I've made the changes, so I'll make a new PR, merge this branch to the master and continue implementing the task

**Comment by rastasheep** [comment]
Cool, btw you don't need to close PR after review, you can push changes to the same branch and PR will be updated automatically

**Comment by MilanBrkic** [comment]
Okay got it, thanks for the tip

**Pull request closed without merging**